tests/test_tags.py

```diff
@@ -923,12 +923,15 @@ def test_default_interpreter(self):
         ]
 
     def test_default_platforms(self, monkeypatch):
-        monkeypatch.setattr(tags, "_platform_tags", lambda: ["plat"])
+        monkeypatch.setattr(tags, "_platform_tags", lambda: iter(["plat", "plat2"]))
```

Comment:
Why change the test to add a second platform? Wouldn't `iter(["plat"])` get the same result for testing the acceptance of an iterable that could be exhausted?

Reply:
Ah, to force the loop through twice. (Sorry, must be more tired than I realized.)

Reply:
Well, you are twice right in a sense, because platforms is the inner loop, but if once in the future this becomes the outer loop, we do would want to force it twice. Making it loop twice makes the test less dependent on the concrete implementation.